pkgs/games/fltrator/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, unzip, fltk, which, libjpeg }:
+
+stdenv.mkDerivation rec {
+  name = "fltrator-${version}";
+  version = "2.3";
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/fltrator/files/v${version}/fltrator-${version}-code.zip/download";
+    sha256 = "125aqq1sfrm0c9cm6gyylwdmc8xrb0rjf563xvw7q28sdbl6ayp7";
+  };
+
+
+  buildInputs = [ fltk unzip which libjpeg ];
+
+  unpackPhase = ''
+  unzip $src
+  '';
+
+  configurePhase = ''
```

Comment:
I think I was unclear with these comments. What I meant was that it should look like this:
```
  configurePhase = ''
    cd fltrator*
    substituteInPlace src/fltrator.cxx\
      --replace 'home += "fltrator/"' "home = \"$out/fltrator/\""
    substituteInPlace src/fltrator-landscape.cxx\
      --replace 'home += "fltrator/"' "home = \"$out/fltrator/\""
    ./configure
  '';
```

That is, the contents of the strings is indented two spaces, but the ending quote is at the same level as the attribute (here, configurePhase)